security/authorization.go

```diff
@@ -184,6 +189,17 @@ func (am *authorizationManager) GetRightsOfCurrentUser(ctx context.Context) map[
 	return rights
 }
 
+func suggestForbiddenError(err error) error {
+	// Caller is suggesting to return a forbidden error except if err is an Unauthorized one
+	switch e := errors.Cause(err).(type) {
+	case errorhandler.DetailedError:
```

Comment:
keycloak-client will be updated in order to return a DetailedError in case of Unauthorized